core/themes/seven/css/style.css

```diff
@@ -1586,6 +1586,25 @@ body.maintenance-page #logo {
   margin: 0 auto;
 }
 
+/* Links for the simplified login pages. */
+.user-form-links ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
```

Comment:
Are you familiar with [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap)?
I almost always use "gap" in combination with flex.